ly/musicxml/lymus2musxml.py

```diff
@@ -92,22 +92,22 @@ def __init__(self):
 
     def parse_text(self, ly_text, filename=None):
         """Parse the LilyPond source specified as text.
-        
+
```

Comment:
This file includes a number of whitespace-only changes in the commit, which you should try to avoid. As I describe in wbsoft/frescobaldi#924 we have the problem that there are many whitespace-only lines in our codebase. It seems you use an editor that automatically removes this upon saving, and while this is generally a good thing (Frescobaldi offers this option too) it triggers "noise" in the commit.

Please configure your editor not to automatically remove such lines.
(This is not meant as criticism. It's a regular issue).